projects/plugins/jetpack/extensions/blocks/revue/edit.js

```diff
@@ -6,7 +7,9 @@ import './view.scss';
 
 // eslint-disable-next-line jsdoc/require-jsdoc
 export default function RevueEdit( { className } ) {
-	const migrationLink = 'https://wordpress.com/go/digital-marketing/migrate-from-revue-newsletter/';
+	const migrationMoreInfoLink =
```

Comment:
Maybe this would be the opportunity to start using `getRedirectUrl` for those links?

Reply:
Ah yes, thank you.